corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -136,15 +136,21 @@ hqDefine('cloudcare/js/utils', [
         }
     };
 
+    var updateScreenReaderNotification = function (notificationText) {
+        $('#sr-notification-region').text("");
```

Comment:
What is the purpose of this?

Reply:
this resets the live region with blank text before updating with the notification text, when the notification text was same it was not getting read by screen-reader on multiple clicks of "Sync" button, this is a fix to that problem.